ArcGISRuntimeSDKQt_CppSamples/Maps/DownloadPreplannedMap/DownloadPreplannedMap.h

```diff
@@ -0,0 +1,92 @@
+// [WriteFile Name=DownloadPreplannedMap, Category=Maps]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef DOWNLOADPREPLANNEDMAP_H
+#define DOWNLOADPREPLANNEDMAP_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class Map;
+class MapQuickView;
+class GraphicsOverlay;
+class PortalItem;
+class OfflineMapTask;
+class PreplannedMapArea;
+class DownloadPreplannedOfflineMapJob;
+class MobileMapPackage;
+class SimpleLineSymbol;
+}
+}
+
+#include <QObject>
+#include <QTemporaryDir>
```

Comment:
we normally alphabetize includes and separate Qt and API headers with a blank line